build/Makefile

```diff
@@ -49,6 +49,7 @@ SUPPRESS_ERROR_OUTPUT ?= 2> /dev/null
 clean: start
 	@echo 'Cleaning...'
 	-@rm -Rf $(BIN_DIR) $(SUPPRESS_ERROR_OUTPUT) ||:
+	-@mkdir -p $(BIN_DIR)/metadata $(SUPPRESS_ERROR_OUTPUT) ||:
```

Comment:
consider calling directory processed_metadata to make a clear distinction to the stuff in src/

Reply:
Changed